tests/rptest/tests/debug_bundle_test.py

```diff
@@ -106,13 +107,16 @@ def test_post_debug_bundle(self, ignore_none: bool):
         res = admin.get_debug_bundle_file(filename=filename, node=node)
         assert res.status_code == requests.codes.ok, res.json()
         assert res.headers['Content-Type'] == 'application/zip', res.json()
+        data_dir = None
         try:
-            data_dir = admin.get_cluster_config(node=node,
-                                                key="debug_bundle_storage_dir")
+            data_dir = admin.get_cluster_config(
```

Comment:
seems like `Admin.get_cluster_config` could be  massaged a bit to avoid this exception handling noise, but so be it.

Reply:
yeah fair, I have a follow up coming shortly and will address in there, just wanted to get things 'green' again asap